app/model/DigitalEdition.scala

```diff
@@ -0,0 +1,25 @@
+package model
+
+import configuration.Config
+
+case class DigitalEdition(id: String, name: String, price: String, campaign: String)
+
+
+object DigitalEdition {
+
+  trait UrlConfig {
+    val externalSubscriptionUrl: String
+  }
+
+  object UK extends DigitalEdition("uk", "UK", "£11.99", "dis_2408")
+  object US extends DigitalEdition("us", "US", "$19.99", "dis_2378")
+  object AU extends DigitalEdition("au", "Australia", "$21.50", "dis_2379")
+  implicit lazy val defaultUrlConfig: UrlConfig = Config
+
+  def getRedirect(edition: DigitalEdition)(implicit urlConfig: UrlConfig): String = {
```

Comment:
You can always have a chat in person for this kind of thing :smile: https://github.com/guardian/recommendations/blob/master/pull-requests.md#talk-is-better

That said: I might be exposing my naivety here, but if we just use the base URL in one method and it doesn't change based on environment does it even need to be in the config?